pages/community/governance/index.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  Hero,
+  HeroImageUtil,
+  Button,
+  Flex,
+  Box,
+  Container,
+  ThickLink,
+  Text,
+} from "@/lib/ui";
+import Head from "next/head";
+import { GovernanceStructure } from "@/components/Governance/GovernanceStructure/GovernanceStructure";
+import { GovernanceFAQ } from "@/components/Governance/GovernanceFAQ/GovernanceFAQ";
+import { DiscordCTA } from "@/components/Governance/DiscordCTA/DiscordCTA";
+import { defineMessages, useIntl } from "react-intl";
+import betta from "../../../assets/heroImages/governance/betta.svg";
+import angler from "../../../assets/heroImages/governance/angler.svg";
+import jelly from "../../../assets/heroImages/governance/jelly.svg";
+
+const messages = defineMessages({
+  heading: {
+    id: "governance.heading",
+    defaultMessage: "Iron Fish Governance",
+  },
+  subheading: {
+    id: "governance.subheading",
+    defaultMessage: "Community-led protocol decisions",
+  },
+  description: {
+    id: "governance.description",
+    defaultMessage:
+      "The Foundation has launched a Discourse website and put out a public call for governance submissions.",
+  },
+  submitProposal: {
+    id: "governance.submitProposal",
+    defaultMessage: "Submit Proposal",
+  },
+  communityDriven: {
+    id: "governance.communityDriven",
+    defaultMessage:
+      "Iron Fish has always been a <underline>community driven project.</underline> Since mainnet launch, our focus has been on how to continue bolstering Iron Fish&apos;s decentralization and resilience.",
```

Comment:
The escaped character is showing up literally for some reason: 

<img width="361" alt="image" src="https://github.com/iron-fish/website/assets/767083/a32122f1-6c2a-4fc2-a65c-f841ec428fdb">